src/cli.rs

```diff
@@ -50,10 +52,15 @@ pub fn parse(target_list: &TargetList) -> Args {
         }
     }
 
+    let dind = env::var("CROSS_DOCKER_IN_DOCKER")
```

Comment:
Call this variable `docker_in_docker`.

Reply:
Also, the environment variable should be documented somewhere.

Reply:
I renamed `dind` to `docker_in_docker` and did add a `Docker in Docker` section to the readme.